apps/filecoin-site/src/app/blog/page.tsx

```diff
@@ -30,10 +30,25 @@ export default async function Blog() {
 
       <PageSection backgroundVariant="light">
         <ul>
-          {posts.map((post) => (
-            <li key={post.slug}>
-              <Link href={`/blog/${post.slug}`}>{post.title}</Link>
-            </li>
+          {blogPostsData.map((post: any) => (
+            <BlogCard
+              key={post.title}
+              title={{ text: post.title }}
+              description={{ text: post.excerpt }}
+              author={post.author}
+              date={post.date}
+              tags={post.categories.map((category: string) => ({
+                text: category,
+              }))}
+              image={{
+                src: post.image.src,
+                alt: post.image.alt,
+              }}
+              cta={{
+                href: `/`,
+                baseDomain: 'https://filecoin.io',
```

Comment:
We have a `BASE_DOMAIN` const if needed but I'm guessing this should be `BLOG.PATH/.....`